PR-DESCRIPTION.md

```diff
@@ -0,0 +1,124 @@
+## Implement Conflict-Free Changelog Management System
+
+### Summary
+
+This PR implements a GitLab-inspired changelog management system that eliminates merge conflicts on `CHANGELOG.md` by using individual YAML files for changelog entries. The system includes automation scripts, multi-server support, and comprehensive documentation.
+
+### What's Changed
+
+**Core System:**
+- ✅ Individual YAML files per changelog entry (timestamped filenames)
+- ✅ JSON schema validation for entry consistency
+- ✅ Three PowerShell automation scripts for the complete workflow
+- ✅ Multi-server support (Azure.Mcp.Server, Fabric.Mcp.Server, etc.)
+
+**Scripts Added:**
+- `eng/scripts/New-ChangelogEntry.ps1` - Interactive generator for creating changelog entries
+- `eng/scripts/Compile-Changelog.ps1` - Compiles YAML files into CHANGELOG.md
+- `eng/scripts/Sync-VsCodeChangelog.ps1` - Syncs main CHANGELOG to VS Code extension CHANGELOG
+
+**Documentation:**
+- `docs/changelog-entries.md` - User guide for contributors
+- `docs/design/changelog-management-system.md` - Design document and implementation details
+- `eng/schemas/changelog-entry.schema.json` - JSON schema for validation
+
+**Infrastructure:**
+- Created `changelog-entries/` folders for Fabric.Mcp.Server and Template.Mcp.Server
+- Created VS Code CHANGELOG.md files for each server
+- Updated CONTRIBUTING.md and AGENTS.md with changelog workflow
+
+### Key Features
+
+**1. No More Merge Conflicts**
+Contributors create individual YAML files instead of editing CHANGELOG.md directly:
```

Comment:
Please update this to tell users the file naming convention: alias-change.yaml

Reply:
I don't like to ask people to name things themselves if we can handle it for them with the helper script.